### ISSUE TITLE
pienv can't detect environment markers from requirements.txt

### ISSUE THREAD

**Issue by aodag** [kennethreitz/pipenv#848]
i have requirements.txt such as below:

```
pyramid ; python_version < '3.0'
```

using pipenv 
```
pipenv install -r requirements.txt --three
```

i got py3 environment and pyramid was installed.
however requirements.txt has environment marker, so pipenv shouldn't install pyramid.

**Comment by kennethreitz** [comment]
this is correct

**Comment by kennethreitz** [comment]
we'd have to look into using the pip parser to do this.

**Comment by techalchemy** [comment]
@kennethreitz @aodag this actually will be fixed in #1095 I believe.
